docker/experiment-covid-native.dockerfile

```diff
@@ -0,0 +1,23 @@
+ARG EXPERIMENTS_VERSION
+
+FROM faasm/experiment-base-native:${EXPERIMENTS_VERSION}
+
+RUN apt update
+
+WORKDIR /code
+RUN git clone https://github.com/faasm/experiment-covid
+
+# Build native 
+WORKDIR /code/experiment-covid/
+RUN git submodule update --init
+RUN ./build/native.sh
+
+# Unzip and copy population files
```

Comment:
This is to allow us running experiments for _any_ country listed in [this directory](https://github.com/faasm/covid-sim/tree/faasm/data/admin_units).